server/integration_tests/explore_test.py

```diff
@@ -491,6 +492,7 @@ def test_fulfillment_nl_size(self):
 
 class ExploreTestEE1(ExploreTest):
 
+  @unittest.skip
```

Comment:
nit: add a TODO here to come back to this test

Reply:
done